Resources/Prototypes/Corvax/human_facial_hair.yml

```
@@ -0,0 +1,15 @@
+﻿- type: marking
+  id: HumanFacialHairHandlebar
+  bodyPart: FacialHair
+  markingCategory: FacialHair
+  sprites:
+  - sprite: Corvax/Mobs/Customization/human_facial_hair.rsi
```

yamllint

[error] wrong indentation: expected at least 3

(indentation)